pkg/server/docs.go

```diff
@@ -65,21 +66,34 @@ var pageTemplate = template.Must(template.New("docs-page").Parse(`
 
 // NewDocsHandler returns a handler func that generates HTML documentation for
 // the given registry.
-func NewDocsHandler(registry *broker.ServiceRegistry) (http.HandlerFunc, error) {
+func NewDocsHandler(router *mux.Router, registry *broker.ServiceRegistry) error {
```

Comment:
Since these are no longer initialization functions, can you refactor these to `Add_Handler`?